sdk/python/pulumi_kubernetes/__init__.py

```diff
@@ -358,6 +311,45 @@
    "kubernetes:coordination.k8s.io/v1beta1:LeaseList": "LeaseList"
   }
  },
+ {
+  "pkg": "kubernetes",
+  "mod": "core/v1",
```

Comment:
This  seems to be the ordering issue I pinged you about. It's now alpha sorted on "$pkg:$mod".